src/main/java/com/gluonhq/substrate/target/DarwinTargetConfiguration.java

```diff
@@ -80,22 +80,7 @@ String getAdditionalSourceFileLocation() {
         return answer;
     }
 
-    private static final List<String> javafxReflectionMacClassList = Arrays.asList(
-            "com.sun.prism.es2.ES2Pipeline",
-            "com.sun.prism.es2.ES2ResourceFactory",
-            "com.sun.prism.es2.ES2Shader",
-            "com.sun.prism.es2.MacGLFactory",
-            "com.sun.scenario.effect.impl.es2.ES2ShaderSource",
-            "com.sun.glass.ui.mac.MacApplication",
-            "com.sun.glass.ui.mac.MacView",
-            "com.sun.glass.ui.mac.MacPlatformFactory",
-            "com.sun.glass.ui.mac.MacGestureSupport",
-            "com.sun.glass.ui.mac.MacMenuBarDelegate",
-            "com.sun.glass.ui.mac.MacCommonDialogs",
-            "com.sun.glass.ui.mac.MacFileNSURL",
-            "com.sun.javafx.font.coretext.CTFactory"
-    );
-
+    private static final String javafxReflectionMacClassList = "reflectionconfig-javafx-x86_64-darwin.json";
```

Comment:
better to create this with the Constants for x86_64 and darwin (e.g. in case we move to AMD64 instead of x86-64)